### PR TITLE
Add Mutual TLS config for Kafka

### PR DESCRIPTION
**Pull request by davidFR**
Related to  #744

Sarama has TLS config for Kafka. I enabled it by adding three parameters to defined CA cert and client cert/key for mutual TLS in Kafka Input and Output : 
` --kafka-tls-ca-cert cacert.cer.pem --kafka-tls-client-cert client.cer.pem --kafka-tls-client-key client.key.pem`

### CODE REVIEW

File: kafka.go [hunk 1]
```diff
@@ -27,6 +31,12 @@ type OutputKafkaConfig struct {
 	UseJSON  bool   `json:"output-kafka-json-format"`
 }
 
+type KafkaTLSConfig struct {
```
[the remark this answers is not in the document]
Comment (davidFR) [reply kafka.go:35]: Corrected

**Comment by davidFR** [comment]
@urbanishimwe  Do you confirm that TestRAWInputIPv4 fails for you on master ?

**Comment by urbanishimwe** [comment]
> @urbanishimwe Do you confirm that TestRAWInputIPv4 fails for you on master ?

yes, it does fail! there is an open PR for its fix.. but don't mind failing tests for now

**Comment by davidFR** [comment]
@arijitAD For testing, I have used the confluent Kafka TLS demo in https://github.com/confluentinc/cp-docker-images/tree/5.3.3-post/examples/kafka-cluster-ssl/
I don't know how to mock Kafka SSL, I'm new to Go Lang, I will see.

**Pull request merged**